i18n/extract.py

```diff
@@ -57,6 +57,20 @@ def add_args(self):
         Adds arguments
         """
         self.parser.description = __doc__
+        self.parser.add_argument(
+            '--merge-js',
+            action='store_true',
+            help='Merge djangojs.po with django.po using msgcat'
+        )
+        self.parser.add_argument(
+            '--overwrite-from-partial',
+            action='store_true',
+            help=(
+                'Overwrite django.po and djangojs.po from (django-partial.po) and (djangojs-partial.po). '
+                'This will also remove (django-saved.po) and (djangojs-saved.po) that are used to restore original '
+                '(django.po) and (djangojs.po) after extract is completed'
+            )
+        )
```

Comment:
Overall the idea is good and the testing is sound.

However, the variable name isn't very clear, so I'm thinking of a different naming like:

```suggestion
        self.parser.add_argument(
            '--combine-partial',
            action='store_true',
            help=(
                'Renames (django-partial.po) and (djangojs-partial.po) to django.po and djangojs.po respectively. '
                'This will also remove (django-saved.po) and (djangojs-saved.po) that are used to restore original '
                '(django.po) and (djangojs.po) after extract is completed. '
                'This is helpful when this `extract` command output is needed to be used immediately '
                ' with no segment/merge workflow in small repositories.'
            )
        )
```